backend/backend/settings/base.py

```diff
@@ -308,7 +313,7 @@ def get_required_setting(
             "USERNAME": REDIS_USER,
             "PASSWORD": REDIS_PASSWORD,
         },
-        "KEY_FUNCTION": "utils.cache_service.custom_key_function",
+        "KEY_FUNCTION": "utils.redis_cache.custom_key_function",
```

Comment:
@muhammad-ali-e will this break any existing functionality?

Reply:
I don't think it breaks, It just move the function from one class/file  to another